Neutaro/assetlist.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "Neutaro",

```

Comment:
Please do not use uppercase letters for chain_name. assetlist.json::chain_name, chain.json::chain_name, and the directory name should all be `neutaro`.

Reply:
Done